src/goto-programs/string_refine_preprocess.h

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************\
+
+Module: Preprocess a goto-programs so that calls to the java String
+        library are recognized by the PASS algorithm
+
+Author: Romain Brenguier
+
+Date:   September 2016
+
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_STRING_REFINE_PREPROCESS_H
+#define CPROVER_GOTO_PROGRAMS_STRING_REFINE_PREPROCESS_H
+
+#include <goto-programs/goto_model.h>
+#include <util/ui_message.h>
+
+class string_refine_preprocesst:public messaget
+{
+ private:
```

Comment:
Please merge the two private sections. I find it confusing when there are several sections. Start with public, then protected and at the end private.